jetty-osgi/test-jetty-osgi/pom.xml

```diff
@@ -576,12 +588,6 @@
           <version>${project.version}</version>
           <scope>test</scope>
         </dependency>
-        <dependency>
-          <groupId>org.conscrypt</groupId>
-          <artifactId>conscrypt-openjdk-uber</artifactId>
```

Comment:
Where did this go?  Is it not used?

Reply:
well actually this dependency (and few others) is declared twice:
- in main dependencies section
- in the java9 profile dependencies section
so as it's really needed we only have to declare it once.
(just figure out we can cleanup more here but will do an other pr)